.github/workflows/github_push.yaml

```diff
@@ -6,7 +6,7 @@ jobs:
     name: MacOS
     strategy:
       matrix:
-        os: [macOS-latest]
+        os: [macOS-15.04]
```

Comment:
Why 15.04?

Reply:
That's the latest version (aka 10.15.04)

Reply:
Oh this is Mac, not Ubuntu. My bad.